src/main/java/org/reaktivity/rym/internal/commands/install/Install.java

```diff
@@ -0,0 +1,193 @@
+/**
+ * Copyright 2016-2019 The Reaktivity Project
+ *
+ * The Reaktivity Project licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package org.reaktivity.rym.internal.commands.install;
+
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+
+import com.github.rvesse.airline.annotations.Command;
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import com.google.gson.JsonArray;
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonParser;
+import com.google.gson.JsonPrimitive;
+import com.google.gson.JsonSyntaxException;
+
+@Command(name = "install",
+    description = "Installs dependencies")
+public final class Install implements Runnable
+{
+    public static final String DEFAULT_DEPENDENCY_FILENAME = "ry.deps";
+    public static final String DEFAULT_DEPENDENCY_LOCK_FILENAME = String.format("%s.lock", DEFAULT_DEPENDENCY_FILENAME);
+
+    public static final String DEFAULT_GROUP_NAME = "org.reaktivity";
+
+    public static final String DEPS_PROP_REPOSITORIES = "repositories";
+    public static final String DEPS_PROP_DEPENDENCIES = "dependencies";
+
+    private final List<String> repositories;
+
```

Comment:
Please remove blank line between final fields.

Reply:
Does this mean just the member variables (i.e. non static), or also remove blank lines between all of the static fields, too?